setup.py

```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name="icepyx",
-    version="0.0.1",
+    version="0.0.2",
```

Comment:
```suggestion
    version="0.3.0",
```

Version string needs to change here :smile: Should this be '0.3.0' or '0.0.3'? I see on https://pypi.org/project/icepyx/ that the last version was '0.0.2'.